firefox-ios/firefox-ios-tests/Tests/XCUITests/ReadingListTests.swift

```diff
@@ -266,19 +266,16 @@ class ReadingListTests: BaseTestCase {
 
     // https://testrail.stage.mozaws.net/index.php?/cases/view/2306993
     // Smoketest
-    func testAddToReaderListOptions() throws {
+    func testAddToReaderListOptions() {
         // Temporarily disabled until url bar redesign work FXIOS-8172
```

Comment:
Nit: remove this comment if it does not apply anymore

Reply:
I can do this in the next PR, so we wont trigger another run for this one if that is ok.